### DISCUSSION TITLE
Hello to All

### DISCUSSION THREAD

**Discussion by yoarora** [cncf/mentoring#941] [Google Summer of Code 2023]
Hello CNCF members,
I am a Computer Engineering student at Purdue University and sent in my proposal for one of the projects listed under CNCF for this year. I am grateful to have @SantiagoTorres, a veteran in CNCF, as my Professor and mentor to help me make meaningful contributions to the CNCF community. I have been learning about CNCF a lot these past couple of weeks and will do my best to contribute to many projects going forward.

**Comment by nate-double-u** [comment]
Welcome, we're glad you are here! Best of luck on your applications 🙂
